debian/changelog

```diff
@@ -1,3 +1,8 @@
+matrix-synapse-py3 (1.3.0) stable; urgency=medium
```

Comment:
I think this needs to be UNRELEASED or something, to avoid confusing the debian release scripts; see https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#debian-changelog for the right incantation